docs/plat/rpi3.rst

```diff
@@ -0,0 +1,297 @@
+ARM Trusted Firmware for Raspberry Pi 3
+=======================================
+
+.. section-numbering::
+    :suffix: .
+
+.. contents::
+
+The Raspberry Pi 3 is an inexpensive single-board computer that contains four
+Cortex-A53 cores, which makes it possible to have a port of the Arm Trusted
+Firmware.
+
+IMPORTANT NOTE: This port isn't secure. All of the memory used is DRAM, which is
+available from both the Non-secure and Secure worlds. This port shouldn't be
+considered more than a prototype to play with and implement elements like PSCI
+to support the Linux kernel.
+
+Design
+------
+
+The SoC used by the Raspberry Pi 3 is the Broadcom BCM2837. It is a strange SoC
```

Comment:
Please remove the word "strange" - someone might take offence.

Reply:
Done.